src/jupyter_contrib_nbextensions/nbextensions/pyVariableInspector/main.js

```diff
@@ -0,0 +1,397 @@
+                "     _getsizeof(var), _trunc(str(var),lenVar), v)",
+                "def var_list():",
+                "    values = _nms.who_ls()",
+                "    _html = '<div class=\"inspector\"><table class=\"table fixed table-condensed table-nonfluid \"><col /> \\",
+                "    <col  /><col /><thead><tr><th >X</th><th >Name</th><th >Type</th><th >Size</th><th >Value</th></tr></thead><tr><td>' + \\",
+                "            '</td></tr><tr><td>'.join([ _tableRow(v) for v in values if (v not in ['_html', '_nms', 'NamespaceMagics', '_ipython']) & (type(eval(v)).__name__ not in types_to_exclude)]) + \\",
+                "            '</td></tr></table></div>'",
+                "    return _html",
+                "print(var_list())"
+            ].join('\n')
+            code_init = code_init.replace('lenName', cfg.lenName).replace('lenType', cfg.lenType)
+            .replace('lenVar', cfg.lenVar)
+            .replace('types_to_exclude',JSON.stringify(cfg.types_to_exclude).replace(/\"/g,"'"))
+            // read configuration     
+        cfg = read_config(cfg, function() {
+            // kernel may already have been loaded before we get here, in which
+            // case we've missed the kernel_ready.Kernel event, so try ctx
+            if (typeof IPython.notebook.kernel !== "undefined" && IPython.notebook.kernel !== null) {
+                if (IPython.notebook.metadata.kernelspec.language.toLowerCase() == 'python') {                    
+                    require(
+                        [
+                            'https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.25.7/js/jquery.tablesorter.min.js'
+                            //'nbextensions/pyVarInspector/colResizable-1.6.min'
+                        ],
+                        function() {
+                            IPython.notebook.kernel.execute(code_init, { iopub: { output: code_exec_callback } }, { silent: false });
+                        })
+                    variable_inspector(cfg, st);
+                } else {
+                    alert("pyVariableInspector only works with a Python kernel")
+                }
+            }
+        }); // called after config is stable  
+
+        // event: if kernel_ready (kernel change/restart)
+        $([IPython.events]).on("kernel_ready.Kernel", function() {
+            IPython.notebook.kernel.execute(code_init, { iopub: { output: code_exec_callback } }, { silent: false });
+            //setTimeout(function() { $('#varInspector table').tablesorter() }, 500)
+                //varInspector_init()
+        })
+
+        // event: on cell execution, update the list of variables 
+        $([IPython.events]).on('execute.CodeCell', varRefresh);
+        $([IPython.events]).on('varRefresh', varRefresh);
+    }
+
+    var create_varInspector_div = function(cfg, st) {
+        function save_position(){
+            IPython.notebook.metadata.pyVarInspector['varInspector_position'] = {
```

Comment:
and given that it's in the `pyVarInspector` object already, it doesn't really need the `varInspector_` prefix for the name, so `IPython.notebook.metadata.pyVarInspector.position`